Specs/Sequence/English/IpAddressModel.json

```diff
@@ -0,0 +1,409 @@
+    {
+      "Text": "0.0.0.0",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "0.0.0.0",
+        "type": "ipv4"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "My PC IP address is 255.255.255.255",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "255.255.255.255",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "255.255.255.255",
+        "type": "ipv4"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "8.8.8.8 is the ip address",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "8.8.8.8",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "8.8.8.8",
+        "type": "ipv4"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "8.8.8.1/24 is the ip address",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "8.8.8.1",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "8.8.8.1",
+        "type": "ipv4"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "08.008.08.08 is the ip address",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "08.008.08.08",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "8.8.8.8",
+        "type": "ipv4"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "My PC IP address is not 256.1.1.1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+  ]
+}
+,
+{
+  "Input": "wrong IP address 1111.1.1.1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+  ]
+}
+,
+{
+  "Input": "wrong IP address 1101.1.1.1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+  ]
+}
+,
+{
+  "Input": "wrong IP address 1.1000.1.1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+  ]
+}
+,
+{
+  "Input": "wrong IP address 1.1.1000.1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+  ]
+}
+,
+{
+  "Input": "wrong IP address 1.1.1.256",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+  ]
+}
+,
+{
+  "Input": "My PC IPv6 address is ABEF:452::FE10",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "ABEF:452::FE10",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "ABEF:452::FE10",
+        "type": "ipv6"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "My PC IPv6 address is 12::1",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "12::1",
+      "TypeName": "ip",
+      "Resolution": {
+        "value": "12::1",
+        "type": "ipv6"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "My PC IPv6 address is ::",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "::",
```

Comment:
This was very interesting. :)
Can you add a code comment with a pointer to an authoritative source that says this is a valid ipv6?
Thanks.